### DISCUSSION TITLE
SONOFF THR320D Elite

### DISCUSSION THREAD

**Discussion by Ratchis** [BernhardSchlegel/BierBot-Bricks#38] [Q&A]
Hi, I wanted to know if it was possible to maintain the temperature display on the screen or if it was possible to implement visible data

thanks

**Comment by BernhardSchlegel** [comment]
Hey Rachis, 

please see this repo https://github.com/BernhardSchlegel/Brick-32 

Best,
Bernhard.

**Reply by Ratchis** [reply]
thanks for the reply
but I can't find an answer to my question of how to keep the screen of my SONOFF THR320D Elite operational

**Reply by BernhardSchlegel** [reply]
So far, it's only on the sourcecode but I did not create a new release. So you have to compile and flash yourself. Let me know if it works afterwards so I can create a release.

https://github.com/BernhardSchlegel/Brick-32/blob/master/docs/DEV.MD
